Target "" links to target "rclcpp::rclcpp" but the target was not found.

Since the last release, a package that depends on cv_bridge started failing with the following error message:

```sh
CMake Error at CMakeLists.txt:17 (add_library):
  Target "ros_image_to_qimage" links to target "rclcpp::rclcpp" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?


CMake Error at /opt/ros/rolling/share/ament_cmake_gtest/cmake/ament_add_gtest_executable.cmake:50 (add_executable):
  Target "test_ros_image_to_qimage_cpp" links to target "rclcpp::rclcpp" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
Call Stack (most recent call first):
  /opt/ros/rolling/share/ament_cmake_gtest/cmake/ament_add_gtest_executable.cmake:37 (_ament_add_gtest_executable)
  /opt/ros/rolling/share/ament_cmake_gtest/cmake/ament_add_gtest.cmake:68 (ament_add_gtest_executable)
  test/CMakeLists.txt:2 (ament_add_gtest)
``` 

Since the other package doesn't use rclcpp at all, I'm going to guess the issue is coming from the changes here.

--- a/cv_bridge/CMakeLists.txt
+++ b/cv_bridge/CMakeLists.txt
@@ -75,6 +75,7 @@ endif()
 ament_export_dependencies(
   OpenCV
   sensor_msgs
+  rclcpp
 )
 
 ament_export_targets(export_${PROJECT_NAME})

